src/pages/CharacterDetails/styles.scss

```diff
@@ -210,11 +210,15 @@
 
     .character-episode-items {
         flex-wrap: wrap;
+        max-height: 132px;
         overflow: hidden;
+        width: calc(100% - 64px);
+        padding: 32px;
+        margin: -32px;
```

Comment:
What is the purpose of these three lines? You can just give horizontal margin.

Reply:
padding is given for box shadow to not clip by its parent and margin is given to fix its position I couldn't find any clever solutions for this to work